force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -71,7 +82,16 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the number is "Positive", "Negative", or "Zero", or null if the number is null.
      */
     public static String checkNumber(Integer a) {
-        return null; // Replace null with the variable you used to store the result
+        if(a == 0){
+            return 'Zero';
+        }
+        else if(a < 0){
+            return 'Negative';
+        }
+        else if(a == null){
+            return null;
+        }
+        return 'Positive'; // Replace null with the variable you used to store the result
```

Comment:
Excellent job. My only tip is that if you are returning from each condition, and this it what I like to do, just make each one an If statement. 

```
    public static String checkNumber(Integer a) {

        if (a == null) {
            return null;
        }
        if (a > 0) {
            return 'Positive';
        }
        if (a < 0) {
            return 'Negative';
        }
            return 'Zero';
    }
```